card/lib/generators/card/card_generator.rb

```diff
@@ -0,0 +1,33 @@
+# -*- encoding : utf-8 -*-
+
+class Card
+  module Generators
+    class CardGenerator < NamedBase
+      include Rails::Generators::Migration
+      extend ActiveRecord::Generators::Migration
+
+      source_root File.expand_path('../templates', __FILE__)
+
+      argument :mount_point, :required=>false
+      argument :prefix, :required=>false
+
+      def self.namespace
```

Comment:
For some reason it wan't to describe this generator as card:card, but you have to invoke as: wagn g card

This makes it show up right in the list (wagn g --help)